src/lib/checkbox/checkbox.md

```diff
@@ -25,8 +25,17 @@ remove the indeterminate state.
 
 ### Click action config
 When user clicks on the `mat-checkbox`, the default behavior is toggle `checked` value and set
-`indeterminate` to `false`. Developers now are able to change the behavior by providing a new value
-of `MatCheckboxClickAction` to the checkbox. The possible values are:
+`indeterminate` to `false`. Developers now are able to change the behavior by
```

Comment:
While updating this, I'd change "Developers now are able to change the behavior by..." to "This behavior can be customized by..."